src/common/pf_lldp.c

```diff
@@ -1283,74 +1482,42 @@ int pf_lldp_parse_packet (
          break;
       case LLDP_TYPE_ORG_SPEC:
       {
-         uint8_t org_id[3];
-         uint8_t org_subtype;
-         pf_get_mem (&parse_info, &offset, sizeof (org_id), org_id);
-         org_subtype = pf_get_byte (&parse_info, &offset);
+         pf_lldp_org_header_t org;
```

Comment:
Typically we put these declaration in the beginning of the function in this stack.

Reply:
Done.